payment_workflow_enterprise/README.rst

```diff
@@ -0,0 +1,16 @@
+===========================
+Payment Workflow Enterprise
+===========================
+
+Set customer payment as default invoice address.
```

Comment:
@yasmineelmrini when we publish a module on github, it is important to explain the meaning of the module. Please add a little more detail.

For example, what is the problem with followups in Odoo 10.0 enterprise? Why do add a new type of partner?